components/api_server/src/model/issue_tracker.py

```diff
@@ -148,7 +151,8 @@ def get_suggestions(self, query: str) -> list[IssueSuggestion]:
         try:
             json = cast(JiraIssueSuggestionJSON, self.__get_json(api_url))
         except Exception as reason:
-            logging.warning("Retrieving issue id suggestions from %s failed: %s", api_url, reason)
+            message = "Retrieving issue id suggestions from %s failed: %s"
+            logger.warning(message, self.suggestions_api % (self.url, "<query redacted>"), reason)
```

Comment:
Did the query include possible secrets? Also, the inline string format this way will probably evaluate on runtime instead of when written to logger; then we might as well use f-string for readability

Reply:
SonarQube started complaining about possible logging injection because the query is user-provided data. I decided that we probably don't need to exact query in the logging anyway, so removed it from the logged message. 

Using a f-string is certainly possible, but since `suggestions_api` is part of a bunch of URLs and endpoints defined as class attributes I don't want to change just one to an f-string.